src/meta/topics/dependencies.md

```diff
@@ -0,0 +1,92 @@
+# Dependencies
+
+There are several dependencies that Node.js relies on to work the way it does.
+
+- [Libraries](#Libraries)
+  - [V8](#V8)
+  - [libuv](#libuv)
+  - [http-parser](#http-parser)
+  - [c-ares](#c-ares)
+  - [OpenSSL](#OpenSSL)
+  - [zlib](#zlib)
+- [Tools](#Tools)
+  - [npm](#npm)
+  - [gyp](#gyp)
+  - [gtest](#gtest)
+
+## Libraries
+
+### V8
+
+The V8 library provides Node.js with a JavaScript engine, which Node.js
+controls via the V8 C++ API. V8 is maintained by Google, for use in Chrome.
+
+- [Documentation](https://v8docs.nodesource.com/)
+
+### libuv
+
+Another important dependency is libuv, a C library that is used to abstract
+non-blocking I/O operations to a consistent interface across all supported
+platforms. It provides mechanisms to handle file system, dns, network, child
+processes, pipes, signal handling, polling and streaming. It also includes a
+thread pool interface for offloading work for some things that can't be done
+asynchronously at the operating system level.
+
+- [Documentation](http://docs.libuv.org/)
+
+### http-parser
+
+HTTP parsing is handled by a lightweight C library called http-parser. It is
+designed to not make any syscalls or allocations, so it has a very small
+per-request memory footprint.
+
+- [Documentation](https://github.com/joyent/http-parser/)
+
+### c-ares
+
+For asynchronous DNS requests, Node.js uses a C library called c-ares.
```

Comment:
If I recall correctly, the DNS module is backed by c-ares, with the exception of getaddrinfo using libuv instead. The rest of core only uses that one getaddrinfo function though, correct?